src/translations/translations.fr.js

```diff
@@ -483,6 +483,69 @@ export const translations = {
   'cc-matomo-info.open-matomo.text': `Vous pouvez accéder à votre Matomo en utilisant votre compte Clever Cloud. Tous les membres de l'organisation peuvent également accéder au service grâce à leur propre compte.`,
   'cc-matomo-info.open-matomo.title': `Accéder à Matomo`,
   //#endregion
+  //#region cc-orga-member-card
+  'cc-orga-member-card.btn.cancel.accessible-name': ({ memberIdentity }) => `Annuler modification du membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.cancel.visible-text': `Annuler`,
+  'cc-orga-member-card.btn.delete.accessible-name': ({ memberIdentity }) => `Supprimer le membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.delete.visible-text': `Supprimer`,
+  'cc-orga-member-card.btn.edit.accessible-name': ({ memberIdentity }) => `Modifier le membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.edit.visible-text': `Modifier`,
+  'cc-orga-member-card.btn.leave.accessible-name': `Quitter l'organisation`,
+  'cc-orga-member-card.btn.leave.visible-text': `Quitter`,
+  'cc-orga-member-card.btn.validate.accessible-name': ({ memberIdentity }) => `Valider la modification du membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.validate.visible-text': `Valider`,
+  'cc-orga-member-card.current-user': `Votre compte`,
+  'cc-orga-member-card.error.last-admin': () => sanitize`<p><strong>Il s'agit du dernier admin de l'organisation</strong></p><p>Veuillez désigner un nouvel admin avant de pouvoir supprimer ce membre.</p>`,
+  'cc-orga-member-card.error.last-admin-is-current-user': () => sanitize`<p><strong>Vous êtes le dernier admin de l'organisation</strong></p><p>Veuillez designer un nouvel admin avant de pouvoir modifier votre rôle ou quitter l'organisation.</p>`,
+  'cc-orga-member-card.mfa-disabled': `2FA désactivée`,
+  'cc-orga-member-card.mfa-enabled': `2FA activée`,
+  'cc-orga-member-card.role.accounting': `Comptable`,
+  'cc-orga-member-card.role.admin': `Admin`,
+  'cc-orga-member-card.role.developer': `Développeur`,
```

Comment:
This is tricky but such a nice catch :+1: :clap: 

Let's try to find the best solution together in sync once I'm done process the rest of UI changes :+1:

Reply:
One of the interesting solutions that came up was to use words to described the activity more than the role: "development", etc.

We decided not to deal with this subject just yet since it's bigger than just this component.